doc/v2.1-joss.md

```diff
@@ -0,0 +1,107 @@
+---
+title: 'khmer release v2.1: software for biological sequence analysis'
+tags:
+    - bioinformatics
+    - sequence analysis
+    - Bloom filter
+    - Count-Min sketch
+    - de Bruijn graph
+    - assembly
+    - graph traversal
+authors:
+    - name: Daniel Standage
+      orcid: 0000-0003-0342-8531
+      affiliation: 1
+    - name: Ali Aliyari
+      orcid:
+      affiliation: 2
+    - name: Lisa J. Cohen
+      orcid:
+      affiliation:
+          - 1
+          - 3
+    - name: Michael R. Crusoe
+      orcid: 0000-0002-2961-9670
+      affiliation: 4
```

Comment:
Is this supposed to be my current affiliation or the affiliation at the time of the contribution?